test/jdk/tools/jlink/plugins/IncludeLocalesPluginTest.java

```diff
@@ -424,47 +414,43 @@ public static void main(String[] args) throws Exception {
 
         helper = Helper.newHelper(isLinkableRuntime);
         if (helper == null) {
-            throw new RuntimeException("Helper could not be initialized");
+            fail("Helper could not be initialized");
```

Comment:
Could be `assertNotNull`, but probably best to not use an assertion in the setup method.

Reply:
Replaced with assertion here, and one other location. I think it is fine